itest/lnd_quiescence_test.go

```diff
@@ -0,0 +1,29 @@
+package itest
+
+import (
+	"github.com/btcsuite/btcd/btcutil"
+	"github.com/lightningnetwork/lnd/lnrpc/devrpc"
+	"github.com/lightningnetwork/lnd/lntest"
+	"github.com/stretchr/testify/require"
+)
+
+// testQuiescence tests whether we can come to agreement on quiescence of a
+// channel. We initiate quiescence via RPC and if it succeeds we verify that
+// the expected initiator is the resulting initiator.
+//
+// NOTE FOR REVIEW: this could be improved by blasting the channel with HTLC
+// traffic on both sides to increase the surface area of the change under test.
+func testQuiescence(ht *lntest.HarnessTest) {
+	alice, bob := ht.Alice, ht.Bob
+
+	chanPoint := ht.OpenChannel(bob, alice, lntest.OpenChannelParams{
+		Amt: btcutil.Amount(1000000),
+	})
+	defer ht.CloseChannel(bob, chanPoint)
+
+	res := alice.RPC.Quiesce(&devrpc.QuiescenceRequest{
+		ChanId: chanPoint,
+	})
+
+	require.True(ht, res.Initiator)
```

Comment:
I think to fully validate it's quiescent we should send a payment and see it fail after this check.